src/util/newconfig/ConfigConstraints.hpp

```diff
@@ -391,8 +313,8 @@ class NumberValueConstraint final : public Constraint {
     [[nodiscard]] std::optional<Error>
     checkValueImpl(Value const& num) const override
     {
-        auto const numValue = std::holds_alternative<double>(num) ? std::get<double>(num) : std::get<int64_t>(num);
-        if (numValue >= min_ && numValue <= max_)
+        auto const numValue = std::get<int64_t>(num);
```

Comment:
Is it never a double anymore?

Reply:
yes; initially I wanted to combine constraints for both a double and int, but there were problems correctly retrieving the value types, so I split it up and have a separate constraint for double. I forgot to remove this part from before :/